test/datahike/test/schema.cljc

```diff
@@ -346,7 +346,7 @@
 (deftest test-update-schema
   (let [cfg "datahike:mem://test-empty-db"
         _ (d/delete-database cfg)
-        _ (d/create-database cfg :initial-schema [name-schema personal-id-schema])
+        _ (d/create-database cfg :initial-tx [name-schema personal-id-schema])
```

Comment:
I was very happy to see that the specs did help catch this typo in this test :blush: